10.0 mig l10n br account banking payment

mileo opened the pull request:
Aguardando https://github.com/OCA/l10n-brazil/pull/618

mileo committed:
[NEW] WIP

fernandomr committed:
WIP alteracoes de visao e adicao de taxa de juros por linha

fernandomr committed:
dados de juros, WIP importacao 400

fernandomr committed:
ajustes do merge, domain e calculo de taxa de mora

fernandomr committed:
sequencia do arquivo e sufixo do arquivo atrelado ao modo de pagamento. numero_documento puxa o name da move_line em vez de apenas numero da fatura

fernandomr committed:
NEW: search for all posted moves

fernandomr committed:
WIP: não busca mais move_lines já exportadas

fernandomr committed:
anotacao para evitar singleton

renatonlima committed:
Change env in travis to work with temp odoo-brazil/odoo

mileo committed:
[REF] oca-autopep8

mileo committed:
[FIX] imports

mileo committed:
[FIX] flake8 errors

hendrixcosta committed:
[REM] Arquivo data obsoleto

hendrixcosta committed:
[FIX] Validacao para campo vazio no legal_name

hendrixcosta committed:
[REM] CAmpos obsoletos

hendrixcosta committed:
[FIX] Dependencias dos modulos

hendrixcosta committed:
[FIX] Flake8

mbcosta committed:
Change license header to OCA standard.

mbcosta committed:
[FIX] Module account_due list, OCA dependecie.

mbcosta committed:
The field BIC were used to inform the 'Codigo de Compesacao' by this reason we need to avoid check of contraint for brazilian banks.

mbcosta committed:
Change deprecated api.one to api.multi.

mbcosta committed:
PEP8 of xml files.

mbcosta committed:
PEP8 - Translation is requeried.

mbcosta committed:
PEP8 - Old api is deprecated.

mbcosta committed:
Remove unused data.

mbcosta committed:
[FIX] Encoding translate file.

mbcosta committed:
Remove unused views.

mbcosta committed:
Remove empty demo data.

mbcosta committed:
Remove inherit fields without change.

mbcosta committed:
Update README file.

mbcosta committed:
PEP8.

mbcosta committed:
[FIX] README

mbcosta committed:
Removed empty line and unuseful variable.

mbcosta committed:
[FIX] Validation of BIC on res.partner.bank .

mbcosta committed:
Included tests to field BIC on Banks.

mbcosta committed:
Remove wrong copyright.

mbcosta committed:
[FIX] Email and name of contributors.

rvalyi commented:
Muito bem, valeu.
Porem accredito que poderiamos aproveitar da migracao para resolver esse problema levantado pelo @mbcosta: https://github.com/OCA/l10n-brazil/issues/588
Ou seja, nao usar mais o campo bic e ter nosso proprio campo pro Brasil. O que acham?
cc @mileo @mbcosta @renatonlima

sadamo committed:
[MIG] Bump module version to 10.0.1.0.0

sadamo committed:
Altera dependência de módulo que não existe na v10.0

sadamo committed:
Rename __openerp__.py to __manifest__.py

sadamo committed:
[MIG] odoo imports

sadamo committed:
Altera nome do modelo pois o modelo pai foi renomeado

sadamo committed:
[MIG] odoo namespace

sadamo committed:
[REF]Alter fields name to match inherited model

sadamo committed:
[FIX]views inherit_id

sadamo committed:
[REM]Herança de wizard nao necessario
O wizard não é mais necessario pois o módulo do qual esse modulo herda
herda já possui suas funcionalidades

sadamo committed:
[REM]Methods already implemented in parent model

mbcosta commented:
Ola @mileo 
Na verdade o modulo l10n_br_account_payment_mode é que depende desse modulo, para continuar esse PR é necessário fazer um "rebase"  se quiserem colocar no repositório do odoo-brasil posso em seguida ver o que faltou na migração.

renatonlima commented:
Replaced by #680
